### ISSUE TITLE
Scalar fields with resolvers don't work

### ISSUE THREAD

**Issue by Jasonoro** [trialspark/graphene-stubs#23]
According to the graphene documentation, [the following is the same](https://docs.graphene-python.org/en/latest/types/scalars/#mounting-scalars):

```python
class Person(graphene.ObjectType):
    name = graphene.String()

# Is equivalent to:
class Person(graphene.ObjectType):
    name = graphene.Field(graphene.String)
````

However, graphene-stubs doesn't allow the above example when using resolvers.

```python
from graphene import ObjectType, String
from graphql import ResolveInfo


class Person(ObjectType):
    name = String()

    @staticmethod
    def resolve_name(_, __: ResolveInfo) -> str:
        return "Test"
```
```
$ mypy test.py
test.py:11: error: No field with name "name" defined
Found 1 error in 1 file (checked 1 source file)
```

When wrapping it in a `Field` it's fine. Is this a known limitation or would this classify as a bug that can be fixed?

**Comment by ckarnell** [comment]
Yeah, as long as that's valid at runtime, this is a bug and there should be no type error. Feel free to make a PR if you'd like to fix! Otherwise I'll try to get to it at some point.

**Comment by matclayton** [comment]
We've just come across this exact issue as well, and its throwing hundreds of errors for us :( I'm not quite sure I understand the MyPy plugin structure or graphene-stubs well enough to make a patch, but I'll see if we have time to. Did anyone else manage to make this work?

Thank you for such a fantastic project!

**Comment by lgnashold** [comment]
Looks like this issue is still present @ckarnell
